subworkflows/local/check_input.nf

```diff
@@ -14,8 +14,15 @@ workflow CHECK_INPUT {
             .splitCsv ( header:true, sep:',' )
             .set { sheet }
 
-        case_info = sheet.first()
-                        .map { create_case_channel(it) }
+        case_info = sheet
+                        .branch { row ->
+                            affected: row.phenotype == "2"
+                            unaffected: row.phenotype == "1"
+                        }
```

Comment:
You might need to add a third category, `unknown` or `missing`. See this  summary on the [ped format](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format)